uaclient/entitlements/fips.py

```diff
@@ -87,9 +87,33 @@ def static_affordances(self) -> "Tuple[StaticAffordance, ...]":
             ),
         )
 
+    def _replace_metapackage_on_cloud_instance(
+        self, packages: "List[str]"
+    ) -> "List[str]":
+        """
+        Identify correct metapackage to be used if in a cloud instance.
+
+        Currently, the contract backend is not delivering the right
+        metapackage on a Bionic Azure or AWS cloud instance. For those
+        clouds, we have cloud specific fips metapackages and we should
+        use them. We are now performing that correction here, but this
+        is a temporary fix.
+        """
+        series = util.get_platform_info().get("series")
+
+        if series != "bionic":
+            return packages
+
+        cloud_id = get_cloud_type()
+        if cloud_id and cloud_id.lower() in ("azure", "aws"):
```

Comment:
Forgot to mention in previous PR on livepatch/fips incompatible_services branch. get_cloud_type() always returns either None or lowercase strings to no need to `lower()` it here.
```suggestion
        if cloud_id and cloud_id in ("azure", "aws"):
```


We could drop that cloud_id.lower() call in _prevent_fips_on_xenial_cloud_instances too